client/src/components/Replies/index.jsx

```diff
@@ -11,14 +11,17 @@ class Replies extends Component {
       this.setState({
         replies: [
           {
+            id: 1,
```

Comment:
@Jamalat-shamallakh I can tell that this wasn't part of your pull request but just FYI - it would be good to move this state out of componentDidMount() and into the initial state for the component.  It's best to avoid calling setState inside componentDidMount, since it causes an additional render as a side effect of the component's first render.